docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -522,11 +522,31 @@ key/value in the same recipe).
 ``xclbin``
 """""""""""""""
 
-This represents a path to a bitstream (FPGA Image) used by this hardware configuration.
-This path must be local to the run farm host that the simulation runs on.
-Only used in Vitis FireSim configurations (an ``agfi`` key/value cannot exist with this
-key/value in the same recipe)
-
+Indicates where the bitstream (FPGA Image) is located, may be one of:
+  * A Uniform Resource Identifier (URI) which specifies a protocol supported either `directly
+    by the fsspec library <https://filesystem-spec.readthedocs.io/en/latest/api.html#built-in-implementations>`_ or
+    by `one of the many third party extension libraries which build on fsspec. <https://filesystem-spec.readthedocs.io/en/latest/api.html#other-known-implementations>`_
+  * A filesystem path available to the manager. Local paths are relative to the `deploy` folder.
+
+Please note that while use use the ``fsspec`` library to handle many different URI protocols, many
+of them require additional dependencies that FireSim itself does not require you to install.
+``fsspec`` will throw an exception telling you to install missing packages if you use one of the
+many URI protocols we do not test.
+
+Likewise, individual URI protocols will have their own requirements for specifying credentials.
+Documentation supplying credentials is provided by the individual protocol implementation.  For
+example:
+* `adlfs for Azure Data-Lake Gen1 and Gen2 <https://github.com/fsspec/adlfs#details>`_
+* `gcfs for Google Cloud Services <https://gcsfs.readthedocs.io/en/latest/#credentials>`_
+* `s3fs for AWS S3 <https://s3fs.readthedocs.io/en/latest/#credentials>`_
+
+For SSH, add any required keys to your ssh-agent.
+
+Please note that while some protocol backendss provide authentication via their own configuration
+files or environment variables (e.g. AWS credentials stored in ``~/.aws``, created by ``aws
+configure``), one can additionally configure ``fsspec`` with additional default keyword arguments
+per backend protocol by using one of the `fsspec configuration
+<https://filesystem-spec.readthedocs.io/en/latest/features.html#configuration>`_ methods.
```

Comment:
Now that we have multiple things that can be a URI, it might be better to move this to a section at the top of this page or it's own page and link to it from anything that can use a URI.